unable to launch on tablet. See error

When i launch on tablet, it crash, see following logs

```
2026-03-26 18:55:09.784  8201-8201  AndroidRuntime          org.github.keepasscompose            E  FATAL EXCEPTION: main (Fix with AI)
                                                                                                    Process: org.github.keepasscompose, PID: 8201
                                                                                                    java.lang.RuntimeException: Unable to instantiate application org.github.keepasscompose.app.KeePassApplication package org.github.keepasscompose: java.lang.ClassNotFoundException: Didn't find class "org.github.keepasscompose.app.KeePassApplication" on path: DexPathList[[zip file "/data/app/~~_soWSPudtlfyWxHf6f7G0g==/org.github.keepasscompose-1dltqSbdUC31JS5V2LsR7A==/base.apk"],nativeLibraryDirectories=[/data/app/~~_soWSPudtlfyWxHf6f7G0g==/org.github.keepasscompose-1dltqSbdUC31JS5V2LsR7A==/lib/arm64, /data/app/~~_soWSPudtlfyWxHf6f7G0g==/org.github.keepasscompose-1dltqSbdUC31JS5V2LsR7A==/base.apk!/lib/arm64-v8a, /system/lib64, /system_ext/lib64]]
                                                                                                    	at android.app.LoadedApk.makeApplicationInner(LoadedApk.java:1478)
                                                                                                    	at android.app.LoadedApk.makeApplicationInner(LoadedApk.java:1403)
                                                                                                    	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:7464)
                                                                                                    	at android.app.ActivityThread.-$$Nest$mhandleBindApplication(Unknown Source:0)
                                                                                                    	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2416)
                                                                                                    	at android.os.Handler.dispatchMessage(Handler.java:107)
                                                                                                    	at android.os.Looper.loopOnce(Looper.java:232)
                                                                                                    	at android.os.Looper.loop(Looper.java:317)
                                                                                                    	at android.app.ActivityThread.main(ActivityThread.java:8705)
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method)
                                                                                                    	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:580)
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:886)
                                                                                                    Caused by: java.lang.ClassNotFoundException: Didn't find class "org.github.keepasscompose.app.KeePassApplication" on path: DexPathList[[zip file "/data/app/~~_soWSPudtlfyWxHf6f7G0g==/org.github.keepasscompose-1dltqSbdUC31JS5V2LsR7A==/base.apk"],nativeLibraryDirectories=[/data/app/~~_soWSPudtlfyWxHf6f7G0g==/org.github.keepasscompose-1dltqSbdUC31JS5V2LsR7A==/lib/arm64, /data/app/~~_soWSPudtlfyWxHf6f7G0g==/org.github.keepasscompose-1dltqSbdUC31JS5V2LsR7A==/base.apk!/lib/arm64-v8a, /system/lib64, /system_ext/lib64]]
                                                                                                    	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:259)
                                                                                                    	at java.lang.ClassLoader.loadClass(ClassLoader.java:637)
                                                                                                    	at java.lang.ClassLoader.loadClass(ClassLoader.java:573)
                                                                                                    	at android.app.AppComponentFactory.instantiateApplication(AppComponentFactory.java:76)
                                                                                                    	at androidx.core.app.CoreComponentFactory.instantiateApplication(CoreComponentFactory.java:51)
                                                                                                    	at android.app.Instrumentation.newApplication(Instrumentation.java:1352)
                                                                                                    	at android.app.LoadedApk.makeApplicationInner(LoadedApk.java:1471)
                                                                                                    	at android.app.LoadedApk.makeApplicationInner(LoadedApk.java:1403) 
                                                                                                    	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:7464) 
                                                                                                    	at android.app.ActivityThread.-$$Nest$mhandleBindApplication(Unknown Source:0) 
                                                                                                    	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2416) 
                                                                                                    	at android.os.Handler.dispatchMessage(Handler.java:107) 
                                                                                                    	at android.os.Looper.loopOnce(Looper.java:232) 
                                                                                                    	at android.os.Looper.loop(Looper.java:317) 
                                                                                                    	at android.app.ActivityThread.main(ActivityThread.java:8705) 
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method) 
                                                                                                    	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:580) 
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:886) 
2026-03-26 18:55:09.791  8201-8201  Process                 org.github.keepasscompose            I  Sending signal. PID: 8201 SIG: 9
```

--- a/androidApp/src/main/AndroidManifest.xml
+++ b/androidApp/src/main/AndroidManifest.xml
@@ -2,7 +2,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
 
     <application
-        android:name=".KeePassApplication"
+        android:name="org.github.keepasscompose.KeePassApplication"
         android:allowBackup="false"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"

